OWNERS

```diff
@@ -0,0 +1,6 @@
+approvers:
+  - cgwalters
+  - debarshiray
+reviewers:
```

Comment:
Contrary to what the Prow docs imply, it's not necessary to duplicate approvers into reviewers.  If you don't want a separate set at all (which I don't think we do right now, since they're the same, we can just drop `reviewers:`).
Example:
https://github.com/openshift/machine-config-operator/blob/master/OWNERS

Reply:
Ok, sure. I had copied [libpod's example](https://github.com/containers/libpod/blob/master/OWNERS), so probably the documentation isn't at fault. :)